attachment_queue/models/attachment_queue.py

```diff
@@ -0,0 +1,106 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import SUPERUSER_ID, api, fields, models, registry
+
+_logger = logging.getLogger(__name__)
+
+
+class AttachmentQueue(models.Model):
+    _name = "attachment.queue"
+    _description = "Attachment Queue"
+    _inherits = {"ir.attachment": "attachment_id"}
+    _inherit = ["mail.thread"]
+
+    attachment_id = fields.Many2one(
+        "ir.attachment",
+        required=True,
+        ondelete="cascade",
+        help="Link to ir.attachment model ",
+    )
+    file_type = fields.Selection(
+        selection=[],
+        help="The file type determines an import method to be used "
+        "to parse and transform data before their import in ERP or an export",
+    )
+    date_done = fields.Datetime()
+    state = fields.Selection(
+        [("pending", "Pending"), ("failed", "Failed"), ("done", "Done")],
+        readonly=False,
+        required=True,
+        default="pending",
+    )
+    state_message = fields.Text()
+    failure_emails = fields.Char(
+        compute="_compute_failure_emails",
+        string="Failure Emails",
+        help="Comma-separated list of email addresses to be notified in case of"
+        "failure",
+    )
+
+    def _compute_failure_emails(self):
+        for attach in self:
+            attach.failure_emails = attach._get_failure_emails()
+
+    def _get_failure_emails(self):
```

Comment:
Could be sent to administrator by default if not other emails set?

Reply:
I think the email notification should stay optional.
For some uncritical attachment, it is possible that we don't want to be polluted by an alert.

Also, one could decide to check the failed attachments once a week for instance and not use the mail alert, etc...
I believe it is easy enough to configure this (on storage backend level) or overridable.

And in a lot of project, the mail behind the admin user is not the one which will check these cases first